R/drive_publish.R

```diff
@@ -40,21 +40,38 @@ drive_change_publish <- function(file,
                                  publish = TRUE,
                                  ...,
                                  verbose = TRUE) {
-  file_update <- drive_is_published(file = file, verbose = FALSE)
-
-  file_update <- confirm_single_file(file_update)
+  file_update <- drive_show_publish(file = file, verbose = FALSE)
+  file_update <- confirm_some_files(file_update)
+  file_update <- split(file_update, 1:nrow(file_update))
```

Comment:
Do you really need to `split()`? Might be a place to use `pmap()`. Or just `map()` on `file$drive_resource`? Consider also if the elevation of publishing info belongs here or down in `change_publish_one()`. If `change_publish_one()` returned one drive file, then a map over files returns a list of drive files, and we can `as_dribble()` it, as done elsewhere. Main point: use a mapping pattern we have elsewhere if can.